openfga/v1/openfga_service.proto

```diff
@@ -849,9 +849,38 @@ message UsersetUser {
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"member\""}
   ];
 }
+message ListUsersUser {
+  string type = 1 [
+    (validate.rules).string = {
+      pattern: "^[^:#@\\s]{1,254}$",
+      ignore_empty: false
+    },
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"group\""}
+  ];
+  oneof user {
```

Comment:
i don't think this will work. a valid response can be `group:b157e743-ae1e-4401-a37b-1526b82b0fa0#member`